ts/user-picker/user-picker.controller.ts

```diff
@@ -271,7 +278,8 @@ module lui.userpicker {
 				"formerEmployees=" + (!!s.showFormerEmployees ? s.showFormerEmployees.toString() : "false") +
 				(!!s.appId && !!s.operations && s.operations.length > 0 ? "&appinstanceid=" + s.appId + "&operations=" + s.operations.join(",") : "") +
 				(!!clue ? "&clue=" + clue : "") +
-				"&searchByEmployeeNumber=" + (!!s.searchByEmployeeNumber ? "true" : "false");
+				"&searchByEmployeeNumber=" + (!!s.searchByEmployeeNumber ? "true" : "false") +
+				(!!s.legalEntityIds && s.legalEntityIds.length > 0 ? "&legalEntityIds=" + s.legalEntityIds.join(",") : "");
```

Comment:
Is the parameter already present in the api (master) because I try to run a query on Lucca instance with legalEntityIds parameters and I got an error.

Reply:
Non, la Pr sera mergé pour la prochaine milestone (19/04/2018).
Le paramètre n'étant pas obligatoire, on peux versionner l'api en l'état.
Ou alors attendre la mise en prod.

Reply:
Ok donc j'attends la release sur Lucca avant de merger car comme le paramètre est envoyé, faudrait pas que ça fasse planter les apps existantes :)

Reply:
(Les apps existantes ne seront pas impacté car le paramètre est optionnel).
La PR à été mergé:
https://github.com/LuccaSA/ilucca/pull/7201